src/test/java/org/opentripplanner/geocoder/bano/BanoGeocoderTest.java

```diff
@@ -27,7 +27,7 @@ public void testOnLine() throws Exception {
 
         boolean found = false;
         for (GeocoderResult result : results.getResults()) {
-            if ("55 Rue du Faubourg Saint-Honoré 75008 Paris".equals(result.getDescription())) {
+            if (result.getDescription().contains("55 Rue du Faubourg")) {
                 double dist = SphericalDistanceLibrary.distance(result.getLat(),
                         result.getLng(), 48.870637, 2.316939);
                 assert (dist < 100);
```

Comment:
I also submitted a PR for this, but I did an additonal cleanup cleanup See PR #2800. I will merge my PR with this after this PR is merged.